ci_tutorial/products_pipeline/multi_product.rst

```diff
@@ -0,0 +1,131 @@
+Products pipeline: multi-product multi-configuration builds
+===========================================================
+
+In the previous section we computed a ``conan graph build-order`` with several simplifications, we didn't take the ``mapviewer`` product into account, and we processed only 1 configuration.
+
+In real scenarios, it will be necessary to manage more than one product and the most common case is that there is more than on configuration for every product. If we build these differents cases sequentially it will be much slower and inefficient, and if we try to build them in parallel there will easily be many duplicated and unnecessary builds of the same packages, wasting resources and even producing issues as race conditions or traceability problems.
+
+To avoid this issue, it is possible to compute a single unified "build-order" that aggregates all the different build-orders that are compute for the different products and configurations.
```

Comment:
```suggestion
To avoid this issue, it is possible to compute a single unified "build-order" that aggregates all the different build-orders that are computed for the different products and configurations.
```